.travis.yml

```diff
@@ -15,7 +15,3 @@ install:
   - pip install -e '.[encryption]'
 
 script: "python setup.py test"
-
-branches:
```

Comment:
Should we remove `.travis.yml` entirely? TravisCI no longer shows up on PRs.

Reply:
Done.